stdlib/SparseArrays/src/linalg.jl

```diff
@@ -27,10 +28,10 @@ function mul!(C::StridedVecOrMat, A::AbstractSparseMatrixCSC, B::Union{StridedVe
 end
 *(A::AbstractSparseMatrixCSC{TA}, x::StridedVector{Tx}) where {TA,Tx} =
     (T = promote_op(matprod, TA, Tx); mul!(similar(x, T, size(A, 1)), A, x, true, false))
-*(A::AbstractSparseMatrixCSC{TA}, B::StridedMatrix{Tx}) where {TA,Tx} =
+*(A::AbstractSparseMatrixCSC{TA}, B::Union{StridedOrTriangularMatrix{Tx},Adjoint{<:Any,<:StridedOrTriangularMatrix{Tx}},Transpose{<:Any,<:StridedOrTriangularMatrix{Tx}}}) where {TA,Tx} =
```

Comment:
Isn't this simply `AdjOrTransStridedOrTriangularMatrix{Tx}`? At least, I can't spot a difference.

Reply:
Yes, you're right. I wanted to put `StridedMatrix{Tx}` as the first element in the union, because this case already got covered by some `*`-method in `matmul.jl` if i remember correctly. I have now replaced this union with `AdjOrTransStridedOrTriangularMatrix{Tx}`.